backend/src/likes/like.entity.ts

```diff
@@ -0,0 +1,21 @@
+
+import {Column, Entity, JoinTable, ManyToMany, PrimaryColumn, PrimaryGeneratedColumn} from 'typeorm';
+import {User} from "../user/user.entity";
+import {Genre} from "../genres/entities/genre.entity";
+
+@Entity()
```

Comment:
@Imadbk7 You cannot have 'like' as a table name since it is a reserved keyword in mysql.